lib/countly.js

```diff
@@ -56,24 +56,27 @@
 }(typeof window !== "undefined" ? window : this, function(Countly) {
     "use strict";
 
-    /** @lends Countly */
-    Countly = Countly || {};
+    // Make sure the code is being run in a browser
+    // eslint-disable-next-line no-undef
+    if (process.browser) {
```

Comment:
Rather than wrapping the entire script in an if, could we change this to an early return? I have to scroll all the way to the bottom of the file (big file!) to see what happens if this test fails.
e.g.

if (!process.browser) { 
  return
}

Reply:
👍🏻